x/deployment/types/events_test.go

```diff
@@ -0,0 +1,174 @@
+package types
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/pkg/errors"
+
+	"github.com/ovrclk/akash/sdkutil"
+)
+
+var (
+	keyAcc = sdk.NewKVStoreKey(auth.StoreKey)
```

Comment:
Interested to know why you are using a `KVStore` key name here?

Reply:
copy-pasta from other test utils for an address string... no good reason.

Have a better suggestion?